rails/config/routes.rb

```diff
@@ -6,6 +6,7 @@
   # root 'welcome#index'
   namespace :api do
     resources :posts
+    resources :analytics
```

Comment:
Don't need all the routes. I believe the syntax is:
```ruby
resources :analytics, only: [:index]
```
http://edgeguides.rubyonrails.org/routing.html